src/ROUTES.js

```diff
@@ -68,6 +68,8 @@ export default {
     getReportRoute: (reportID) => `r/${reportID}`,
     REPORT_WITH_ID_DETAILS_SHARE_CODE: 'r/:reportID/details/shareCode',
     getReportShareCodeRoute: (reportID) => `r/${reportID}/details/shareCode`,
+    REPORT_ATTACHMENT: 'r/:reportID/attachment',
+    getReportAttachmentRoute: (reportID, source) => `r/${reportID}/attachment?source=${encodeURI(source)}`,
```

Comment:
I think I can remove the changes here and in `linking` which would disable deep linking, but also prevent me from using the global `Navigation` object in `ImageRenderer` _(I'll post a similar note there)_

Right now if we refresh the page or copy paste the URL while we have the attachment carousel open, it would load the page with the attachment carousel open 

The question is, do we want to deliberately not allow this? It can be used as a feature linking to specific attachments